app/views/candidate_mailer/new_offer_made.text.erb

```diff
@@ -8,10 +8,14 @@ Congratulations! You have an offer from <%= @provider_name %> to study <%= @cour
  - <%= condition.text %>
 <% end %>
 
-Contact <%= @provider_name %> if you have any questions about this.
+You will have to pass essential checks before you can start your training.
 
-<% if @show_deadline_reminder %>
-  If you want to accept this offer, you must do so by <%= I18n.l(CycleTimetable.decline_by_default_date.to_date, format: :no_year) %>. If you have not responded by then, the offer will be automatically declined on your behalf.
-<% end %>
+These are:
+  - An enhanced disclosure and barring service (DBS) check. This is a criminal records check to make sure it is safe for you to work with children. If you are from outside of the UK and Ireland then the training provider will request a criminal records check from your home country. A fitness to train to teach check.
+  - A fitness to train to teach check. These are questions to check your ability to meet teaching standards, both physically and mentally.
```

Comment:
These are default selected offer conditions but not mean all offer will have this conditions. When provider makes an offer those conditions could be uncheck and this content could not be valid. Can you talk with content designer about it, please?